### ISSUE TITLE
Error when building on macOS

### ISSUE THREAD

**Issue by alokdhir** [xqemu/xqemu#239]
Mojave v 10.14.5
```
[adhir@bigmac:~/src/xqemu] sh build.sh 
Compiling for MacOS…
readlink: illegal option -- f
usage: readlink [-n] [file ...]
+ ./configure '--extra-cflags=-DXBOX=1 -O0 -g -march=native ' --enable-debug --disable-cocoa --target-list=i386-softmmu --enable-trace-backends=nop --enable-sdl --with-sdlabi=2.0 --disable-curl --disable-vnc --disable-vnc-sasl --disable-docs --disable-tools --disable-guest-agent --disable-tpm --disable-live-block-migration --disable-rdma --disable-replication --disable-capstone --disable-fdt --disable-libiscsi --disable-spice --disable-user --disable-stack-protector --disable-glusterfs --disable-bluez --disable-gtk --disable-curses --disable-gnutls --disable-nettle --disable-gcrypt --disable-crypto-afalg --disable-virglrenderer --disable-vhost-net --disable-vhost-crypto --disable-vhost-vsock --disable-vhost-user --disable-virtfs --disable-libssh2 --disable-snappy --disable-bzip2 --disable-vde --disable-libxml2 --disable-seccomp --disable-numa --disable-lzo --disable-smartcard --disable-usb-redir

ERROR: glib-2.40 gobject-2.0 is required to compile QEMU
```

**Comment by dracc** [comment]
Looks like you haven't installed glib and gobject as instructed on the [Building](http://xqemu.com/developers/building/) page?

**Comment by JayFoxRox** [comment]
@GXTX said on [XQEMU developer Discord (XboxDev)](https://xqemu.com/#developers), that the same error happens on CI when updating homebrew (See https://travis-ci.org/GXTX/xqemu/jobs/549266906).

This is from arch-linux, where compilation works with glib 2.60.2-1:

```
$ pkg-config --atleast-version=2.40 gobject-2.0; echo $?
0
```
```
$ pkg-config --libs --cflags gobject-2.0
-I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/lib/libffi-3.2.1/include -lgobject-2.0 -lglib-2.0 
```
```
$ pkg-config --atleast-version=2.40 gthread-2.0; echo $?
0
```
```
$ pkg-config --libs --cflags gthread-2.0
-pthread -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -pthread -lgthread-2.0 -lglib-2.0 
```

Please also check which version of glib is installed.

Even the QEMU 3.1.0 update doesn't seem to touch the configure script.
So I think his might be a problem in how homebrew packages glib?

**Comment by JayFoxRox** [comment]
Will hopefully be resolved by #243

**Comment by GXTX** [comment]
@alokdhir can you please test #243?
